.gitlab-ci.yml

```diff
@@ -488,7 +467,7 @@ build/nocuda/clang/core/release/shared:
 
 build/nocuda/intel/core/debug/shared:
   <<: *default_build_with_test
-  image: localhost:5000/gko-nocuda-gnu9-llvm8
+  image: localhost:5000/add_cuda11_container_1b1857a4_gko-nocuda-gnu9-llvm8
```

Comment:
Question: Are these changes necessary or left-over from the cuda11 branch?

Reply:
They are necessary, I had to upgrade the intel compiler to 2020 for this PR to work properly. I used the CUDA 11 PR to update this at the same time. The Intel 2019 has some incompatibility with GNU 9 backend when considering C++14 code. Therefore we will need to merge the CUDA11 container before this.